GoogleForCreators/web-stories-wp
# FTUE: Animated Navigator

## Context
Part of creating the requested animations for the help center. This is purely a storybook component for now. Unopinionated on the navigation structure or its children

## Summary
Adds animated markup & styles to the help center navigator

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
NA

## Testing Instructions
Go to storybook -> story editor -> components -> help center -> navigator -> see that when you click the `toggle children` button the navigator animates to the height of the new child and transitions the bottom nav bar in/out of view

### QA






### UAT







## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Partially Fixes #5884
